src/features/user/UserProfile/components/UserShareAndSupport.tsx

```diff
@@ -88,35 +89,56 @@ export default function UserShareAndSupport({ userprofile }: any) {
           <SocialShareContainer userprofile={userprofile} />
         </motion.div>
       )}
+
       <div className={styles.bottomIconsRow}>
+
         {userprofile.type !== 'tpo' && (
           <div
-            className={styles.iconTextColumnSupport}
+            className={styles.iconTextColumn}
             onClick={() => {
               router.push(`/s/${userprofile.slug}`);
             }}
           >
-            <div className={styles.bottomIconBgSupport}>
-              <Support width="37px" paddingLeft="10px" />
-              <p className={styles.bottomRowTextSupport}>{t('me:support')}</p>
+            <div className={styles.bottomIconBg}>
+              <Support width="39px" />
             </div>
+            <p className={styles.bottomRowText}>{t('me:support')}</p>
           </div>
-         )}
-        <div className={styles.iconTextColumnSupport}>
+        )}
+
+        {userprofile.url && (
+          <a
+            className={styles.iconTextColumn}
+            href={userprofile.url}
```

Comment:
This does not work, e.g. http://localhost:3000/t/www.epublica.de for my test profile:
<img width="712" alt="Bildschirmfoto 2020-12-04 um 13 51 47" src="https://user-images.githubusercontent.com/1532418/101166060-3e9f6980-366a-11eb-9598-6f781b182bc0.png">

Also this is regarded as an unsafe link!

Reply:
Happening when users do not explicitly add http:// or https:// for their websites:

<img width="459" alt="Bildschirmfoto 2020-12-04 um 13 55 29" src="https://user-images.githubusercontent.com/1532418/101166326-b1104980-366a-11eb-98dc-f2194baf4cb0.png">

Reply:
To solve the security issue just add `rel="noopener noreferrer" `